Scripts/Reset-LocalAdministratorPassword.ps1

```diff
@@ -27,15 +27,15 @@
 Param
 (
     #Encryption key.
-    [parameter(Mandatory=$true)][string]$SecretKey = "Global-Encryption-Key",
+    [parameter(Mandatory = $true)][string]$SecretKey = "Global-Encryption-Key",
```

Comment:
This will break the build script, you need to branch off dhruvb14/master again then attempt to make changes for this file and all others.